sass/README.md

```diff
@@ -101,7 +100,7 @@ Comprehensive example of ordering items within a declaration block:
   }
 
   &::before {
-    content: "";
+    content: '';
```

Comment:
I think house style is double quotes for `content`, as it's still a CSS property.

Reply:
_prettier_